asyncapi/conference-website
# feat: add a gallery section #264

fix issue - #264 

## What does this PR do?
   This PR addresses the issue #264 by adding a new feature of gallery section on website .

## Screen Recording of visual changes 
[screen-3.webm](https://github.com/user-attachments/assets/294faa2c-6ae5-446a-a332-d520bae6ec65)
[gallery-4.webm](https://github.com/user-attachments/assets/c3fc4740-acee-4e6a-b60e-4cedf6e88d10)


## Type of Changes
- feature added

## How should this be tested ?
1.Go to Homepage
2.Click on Gallery of Header
3.It will redirect to Gallery section.
4.Click Button "Browse All"
5. It will redirect to Google Drive where you can see all photos.
